### PR TITLE
Upgrade freckle-app

### PR DESCRIPTION
**Pull request by eborden**
`freckle-app` has been upgraded to include more stats collection. We
want to use this in `fancy-api` and `tts`.

### CODE REVIEW

File: lts/18/13/FR2.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# https://renaissancelearning.atlassian.net/wiki/spaces/EN/pages/41987178508/Shared+Backend+Stackage+Snapshot
+resolver: lts-18.13
+
+packages:
+  # === Dependencies we own
+
+  # Newer versions that will arrive in next major LTS
+  - faktory-1.1.2.0@sha256:b2a1986095aefa645f6ad701504e1671ddfeb36f11b68434837fc9fcd3594ca8,9078
+  - aws-xray-client-persistent-0.1.0.2@sha256:f00b3c3da576c488f2605ab5d049437d935eae429e7fe0eb9a6dc53d0367aebc,1682
+  - freckle-app-1.0.1.0@sha256:6ca3605957cc137cd8127d970b79e71975d07a4707e8167fbf1fb84b8b420729,6379
```
Comment (eborden) [inline lts/18/13/FR2.yaml:10]: The change

**Review by pbrisbin — APPROVED** [review]

**Comment by eborden** [comment]
Most of the change in this LTS is preperation for GHC 9 and Aeson 2.

**Comment by stackptr** [comment]
Why `18.15` and not `18.14`? And why is the recently created snapshot moved versus creating a new snapshot?

**Comment by eborden** [comment]
- `18.15` is the latest.
- This is not a moved snapshot. I did move the one I just created in this commit stream, as it has not been merged yet.

**Comment by stackptr** [comment]
Ah yeah I got confused looking at a single commit

**Review by stackptr — APPROVED** [review]

**Pull request merged**